Glyssen/Utilities/CustomColor.cs

```diff
@@ -57,6 +67,28 @@ public virtual Color GetMatchColor(int i)
 		{
 			return MaxContrastColorList[i % MaxContrastColorList.Count];
 		}
+
+		public virtual Color GetForeColorByCharacter(AssignCharacterViewModel.Character character)
+		{
+			if (character.CharacterId== "Jesus")
```

Comment:
I checked out this branch and opened a project in Glyssen (because I wanted to see if there would be obvious color contrast problems). I get a NullReferenceException here (character == null) in Luke 2:23-38 in my Menya project. (I e-mailed the crash report directly to @andrew-polk.)